### ISSUE TITLE
v1.8.12: Firmware update check fails with `this.warning is not a function`

### ISSUE THREAD

**Issue by agross** [ioBroker/ioBroker.zigbee#1779]
This is logged while performing the update check with v1.8.12. `0x001788010d1988d7` had an update available that I could install with v1.8.10 after downgrading the adapter.

```
2023-07-04 21:52:52.254  - info: zigbee.0 (443917) Failed to check if update available for '0x0017880109c21225' (this.warning is not a function)
2023-07-04 21:52:54.670  - info: zigbee.0 (443917) Failed to check if update available for '0x0017880109c59798' (this.warning is not a function)
2023-07-04 21:52:54.679  - info: zigbee.0 (443917) Failed to check if update available for '0x001788010c66e9e7' (this.warning is not a function)
2023-07-04 21:52:54.701  - info: zigbee.0 (443917) Failed to check if update available for '0x001788010c66ea0c' (this.warning is not a function)
2023-07-04 21:52:56.742  - info: zigbee.0 (443917) Failed to check if update available for '0x001788010c4eec3d' (this.warning is not a function)
2023-07-04 21:52:56.754  - info: zigbee.0 (443917) Failed to check if update available for '0x001788010c4d5bb2' (this.warning is not a function)
2023-07-04 21:52:56.762  - info: zigbee.0 (443917) Failed to check if update available for '0x001788010c4eebe5' (this.warning is not a function)
2023-07-04 21:52:56.850  - info: zigbee.0 (443917) Failed to check if update available for '0x001788010b3f1fe9' (this.warning is not a function)
2023-07-04 21:52:57.045  - info: zigbee.0 (443917) Failed to check if update available for '0x001788010d1f63fb' (this.warning is not a function)
2023-07-04 21:52:57.071  - info: zigbee.0 (443917) Failed to check if update available for '0x001788010d1988d7' (this.warning is not a function)
2023-07-04 21:53:03.061  - info: zigbee.0 (443917) Failed to check if update available for '0x54ef44100045cc23' (this.warning is not a function)
2023-07-04 21:53:23.732  - info: zigbee.0 (443917) Failed to check if update available for '0x50325ffffe784bdf' (Device didn't respond to OTA request)
```

**Comment by arteck** [comment]
check GIT version

**Comment by agross** [comment]
Closed by 8f192a20efb56e0d47384274446132cfebbd5f9d
